composer.json

```diff
@@ -18,7 +18,7 @@
         "ext-openssl": "*",
         "ext-reflection": "*",
         "fightbulc/moment": "^1.26",
-        "firebase/php-jwt": "^5.0",
+        "firebase/php-jwt": "^6.0",
```

Comment:
How are we planning to take care of breaking changes mentioned at https://github.com/firebase/php-jwt/releases/tag/v6.0.0

Reply:
From the breaking changes list, we are only using JWT::encode and the syntax used is as mentioned in the breaking changes list i.e. we are passing the third argument.